client/src/components/FeatureCalloutWidget.tsx

```diff
@@ -71,7 +87,10 @@ const FeatureCalloutWidget = withI18n()((props: withI18nProps) => {
           focusTrapOptions={{
             clickOutsideDeactivates: true,
             returnFocusOnDeactivate: false,
-            onDeactivate: () => setWidgetVisibility(false),
+            onDeactivate: () => {
+              amplitude.getInstance().logEvent("closeFeatureCalloutWidget");
```

Comment:
Since we have more events now, I'm thinking about the best way to standardize this (e.g. a WOW-specific amplitude wrapper where you can just call `logEvent(eventType)` where `eventType` is some big list of strings). That way you won't accidentally pass a misspelled string and have 2 different event types going